addons/resource.language.en_gb/resources/strings.po

```diff
@@ -21588,3 +21588,40 @@ msgstr ""
 msgctxt "#39113"
 msgid "Center Mix Level in dB relative to metadata or default (-3 dB)"
 msgstr ""
+
+#: xbmc/filesystem/VideoDatabaseDirectory/DirectoryNodeMoviesOverview.cpp
+#: xbmc/filesystem/MusicDatabaseDirectory/DirectoryNodeOverview.cpp
+#: xbmc/filesystem/VideoDatabaseDirectory/DirectoryNodeTvShowOverview.cpp
+msgctxt "#39300"
+msgid "Types"
+msgstr ""
+
+#: xbmc/video/dialogs/GUIDialogVideoInfo.cpp
+msgctxt "#39301"
+msgid "Manage movie version"
+msgstr ""
+
+#: xbmc/video/dialogs/GUIDialogVideoInfo.cpp
+msgctxt "#39302"
+msgid "Convert movie to an additional version of moive"
```

Comment:
s/moive/movie ;)

Reply:
I'll rephrase this. I tried several times, but none of them seems good enough, do you have any suggestion?

Reply:
Wouldn't `Add additional version` or `Add another version` suffice? (I like the second one better I think)

Not having tested this, I assume this string would only show up when the user is already in the context of a movie.